README.md

```diff
@@ -27,3 +27,15 @@ Scenario: Remaining cards correctly updated after drawing
 - Draw cards from the deck 5 times
 - Draw between 1 and 5 cards from the deck each time.
 - Verify that 52-[total cards drawn] remain in the deck at the end of the test
+
+# Implementation:
+
+The branch 'mantunes' contains a solution in C# created in Visual Studio 2017. It uses Nunit and Specflow to complete the scenario requested.
+C# was selected as it is the language I have more exposure at this moment.
+Specflow was used as its big advantage is to create and maintain very easily readable tests.
+The solution contains a feature file, its steps and a POCO model.
```

Comment:
Ellaborate on acronyms, it is a good practice to get into as they can often be mis-understood due to the need for assumption on its meaning.